django_celery_results/apps.py

```diff
@@ -12,4 +12,4 @@ class CeleryResultConfig(AppConfig):
     name = 'django_celery_results'
     label = 'django_celery_results'
     verbose_name = _('Celery Results')
-    default_auto_field = 'django.db.models.AutoField'
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
I'm curious, if this change is backward compatible?

Reply:
I _think_ it's backwards compatible. It's changing the DB type from an integer field to a 64-bit integer, increasing its range. 

However, I have a few concerns for users of the library:

- I'm not sure if there are any performances considerations when applying this migration on a table with a large number of rows. Would the `migrate` script takes a while to run? What kind of locks does it need? How does it varies between databases? 
- The django docs have [a large note](https://docs.djangoproject.com/en/5.0/ref/settings/#std-setting-DEFAULT_AUTO_FIELD) about the change, and if folks have a many to many field that link to one of the tables, they'll need to do an extra migration:
   > Unfortunately, the primary keys of existing auto-created through tables cannot currently be updated by the migrations framework.

So it might be worthy of a major bump to signal these pitfalls...